iphone/Classes/TiUIWebView.m

```diff
@@ -540,6 +540,13 @@ -(void)setHandlePlatformUrl_:(id)arg
     willHandleUrl = [TiUtils boolValue:arg];
 }
 
+-(void)setUserAgent_:(id)args
+{
+    ENSURE_TYPE(args, NSString);
+    NSDictionary *dict = @{@"UserAgent":args};
```

Comment:
That's I meant that is should be allowed (e.g. by checking the argument type) :-) Check the Android implementation - if the reset functionality is not exposed there, it's ok for now to skip it.